gsa/src/web/pages/nvts/listpage.js

```diff
@@ -50,50 +72,195 @@ export const ToolBarIcons = () => (
   />
 );
 
-const Page = ({filter, onFilterChanged, onInteraction, ...props}) => (
-  <React.Fragment>
-    <PageTitle title={_('NVTs')} />
-    <EntitiesPage
-      {...props}
-      createFilterType="info"
-      dashboard={() => (
-        <NvtsDashboard
+const fallbackFilter = Filter.fromString('sort-reverse=created');
+
+const NvtsPage = () => {
+  // Page methods and hooks
+  const [downloadRef, handleDownload] = useDownload();
+  const [, renewSessionTimeout] = useUserSessionTimeout();
+
+  // Powerfilter
+  const [filter, isLoadingFilter] = usePageFilter('nvt', {fallbackFilter});
+  const prevFilter = usePrevious(filter);
+  const simpleFilter = filter.withoutView();
```

Comment:
Should be removed because it isn't necessary anymore see https://github.com/greenbone/gsa/pull/2795
```suggestion
```